src/app/components/directivas-estructurales/directivas-estructurales.component.html

```diff
@@ -0,0 +1,46 @@
+<div>
+    <h3>Exercise 2</h3>
+    <h4>NgFor</h4>
+    <ul>
+        <li *ngFor="let i of array">{{i}}</li>
+    </ul>
+    <h4>NgIf</h4>
+    <ul>
+        <span *ngFor="let i of array"><li *ngIf="i!=='B'">{{i}}</li></span>
```

Comment:
En algunos casos, estás utilizando envolturas <span> innecesarias alrededor de las directivas estructurales. Puedes utilizar <ng-container> en su lugar.
       <ng-container *ngFor="let i of array">
            <li *ngIf="i!=='B'">{{ i }}</li>
        </ng-container>

Reply:
Lo hice de esa manera por que en la descripcion no especifica si esta permitido en uso de container o template. Por eso lo hice dos veces en la parate de abajo lo hice con ng-template que es muy similar a como me recomiendas.

![image](https://github.com/josago11/Apex_Angular_Bootcamp/assets/41796207/ae0840f8-7ac1-4c76-9d80-639aa7f6e1cf)

Reply:
done